docs/source/topics/processes/usage.rst

```diff
@@ -728,9 +727,7 @@ The 'scheduled' indicates that the actual killing might not necessarily have hap
 This means that even after having called ``verdi process kill`` and getting the success message, the corresponding process may still be listed as active in the output of ``verdi process list``.
 
 By default, the ``pause``, ``play`` and ``kill`` commands will only ask for the confirmation of the runner that the request has been scheduled and not actually wait for the command to have been executed.
-This is because, as explained, the actual action being performed might not be instantaneous as the runner may be busy working with other processes, which would mean that the command would block for a long time.
-If you want to send multiple requests to a lot of processes in one go, this would be ineffective, as each one would have to wait for the previous one to be completed.
-To change the default and actually wait for the action to be completed and await its response, you can use the ``--wait`` flag.
```

Comment:
This might be phrased a bit verbose, but I think completely removing it is losing some information. I think it is important to get across that if you wait, it might take long depending on how busy the worker is. This would be important information for a user wondering why the worker seems to not be responding. Could you restore this in a more succinct manner perhaps?

Reply:
I'll give it a look. I felt that most of this was already said in other word in previous paragraphs. I planned to give this a more thorough read anyway (that's why I put this at draft) since there is a fair amount of repetitive info in this section imo.

Thanks for taking a look! :pray: